integration-libs/cdc/components/register/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './facade/index';
```

Comment:
Instead of having folder `/facade` here, let's create 2 folders:
/register
/forgot-password
and let's move there relevant modules and components + create index.ts files there

Then in this file, let's do:
```suggestion
export * from './forgot-password/index';
export * from './register/index';
```

Reply:
okay Kris, will do so